docs/entities/renderer.md

```diff
@@ -122,17 +123,17 @@ graph LR;
     LivingEntityRenderer-->MobRenderer;
     MobRenderer-->AgeableMobRenderer;
     AgeableMobRenderer-->HumanoidMobRenderer;
-    LivingEntityRenderer-->PlayerRenderer;
+    LivingEntityRenderer-->AvatarRenderer;
     
     class EntityRenderer,AbstractBoatRenderer,AbstractMinecartRenderer,ArrowRenderer,LivingEntityRenderer,MobRenderer,AgeableMobRenderer,HumanoidMobRenderer red;
-    class ArmorStandRenderer,PlayerRenderer blue;
+    class ArmorStandRenderer,AvatarRenderer blue;
 ```
 
 - `EntityRenderer`: The abstract base class. Many renderers, notably almost all renderers for non-living entities, extend this class directly.
 - `ArrowRenderer`, `AbstractBoatRenderer`, `AbstractMinecartRenderer`: These exist mainly for convenience, and are used as parents for more specific renderers.
 - `LivingEntityRenderer`: The abstract base class for renderers for [living entities][livingentity]. Direct subclasses include `ArmorStandRenderer` and `PlayerRenderer`.
 - `ArmorStandRenderer`: Self-explanatory.
-- `PlayerRenderer`: Used to render players. Note that unlike most other renderers, multiple instances of this class used for different contexts may exist at the same time.
+- `AvatarRenderer`: Used to render avatars, such as players. Note that unlike most other renderers, multiple instances of this class used for different contexts may exist at the same time.
```

Comment:
Since I assume there's only PlayerRenderer and MannequinRenderer extending this, add those two to the list and the diagram as well.